cmp/grid/Grid.js

```diff
@@ -763,7 +763,7 @@ class GridLocalModel extends HoistModel {
 
     processCellForClipboard = ({value, node, column}) => {
         const {model} = this,
-            recId = node.id,
+            recId = node.data.id,
```

Comment:
`node.data` could potentially be null/undefined here for group rows, I think the code was guarding against this by getting the id and looking up the record below. We should early-out if the node has no data, and just use the `node.data` as the record below to avoid the store lookup